docs/data-sources/vrf_fallback_route_group.md

```diff
@@ -0,0 +1,68 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Networking"
+layout: "aci"
+page_title: "ACI: aci_vrf_fallback_route_group"
+sidebar_current: "docs-aci-data-source-aci_vrf_fallback_route_group"
+description: |-
+  Data source for Vrf Fallback Route Group
+---
+
+# aci_vrf_fallback_route_group #
+
+Data source for Vrf Fallback Route Group
```

Comment:
Logic for `{{getResourceNameAsDescription .ResourceName}}` should be adjusted. getResourceNameAsDescription now creates title case from resource name change vrf into Vrf, which should be VRF. Likely we need some overwrite mechanism here, that replaces Vrf into VRF.

```suggestion
Data source for VRF Fallback Route Group
```